### ISSUE TITLE
Can I disable the sshfs's cache function?

### ISSUE THREAD

**Issue by chent11** [abiosoft/colima#338]
The issue I reported in https://github.com/lima-vm/lima/issues/905. When I use docker as a compiling environment, this is annoying. I know how to disable this cache function in lima, but it seems I can't do that in colima.

**Comment by abiosoft** [comment]
At the moment, no. However you may be able to use Lima override config.

**Comment by chent11** [comment]
Thanks for this suggestion, but it seems when I restart the colima vm, the config would be overridden again by colima, which means I need to use lima to override the config every time I start the colima vm, am I right?

**Comment by abiosoft** [comment]
Yeah, Colima generates the config on every start but Lima overrides take precedence.

You would create a `~/.lima/_config/override.yaml` file and specify the mounts as you would in a Lima config file.
